packages/web/stores/assets/utils.ts

```diff
@@ -1,15 +1,19 @@
 import { Buffer } from "buffer";
 import { sha256 } from "sha.js";
 
+/**
+ *
+ * @param path - The path of the IBC transfer. i.e. "channel-2/transfer/channel-1/..." or just the channel: "channel-1"
+ * @param coinMinimalDenom Coin denom on source chain.
+ * @returns IBC hash of the coin denom.
+ */
 export function makeIBCMinimalDenom(
-  sourceChannelId: string,
+  path: string,
   coinMinimalDenom: string
 ): string {
   return (
     "ibc/" +
-    Buffer.from(
-      sha256_fn(Buffer.from(`transfer/${sourceChannelId}/${coinMinimalDenom}`))
-    )
+    Buffer.from(sha256_fn(Buffer.from(`transfer/${path}/${coinMinimalDenom}`)))
       .toString("hex")
       .toUpperCase()
   );
```

Comment:
If we place the earlier function into `@osmosis-labs/utils`, can we replace this with it?